birdhouse/components/proxy/default.env

```diff
@@ -6,6 +6,12 @@ export PROXY_IMAGE="nginx:1.23.4"
 # Any WPS processes taking longer than this should use async mode.
 export PROXY_READ_TIMEOUT_VALUE="240s"
 
+# CORS settings for the proxy
+# Set to "*" to allow all origins by default
+# If you want to restrict access, set this to the space-delimited allowed domain(s)
+# Can use other variables if needed (e.g.: '${BIRDHOUSE_PROXY_SCHEME}://${BIRDHOUSE_FQDN_PUBLIC}')
+export BIRDHOUSE_PROXY_CORS_ALLOW_ORIGIN="*"
```

Comment:
One very small last thing:

Can we add a note here that reminds the user that if this is unset (or set to the empty string) then it will default to "*" (because of the if block in cors.include). 

That might be something that would surprise users if they haven't looked too closely at cors.include

Reply:
On the other hand, if the user explicitly override it to empty string in `env.local`, maybe he wants to **not** allow '\*' and if we still set to '\*' in `cors.include`, is that a good thing?!

Is there any scenarios where the user really want that value to be empty string and we should not block that?